README.md

```diff
@@ -63,6 +63,12 @@ Our app helps (aspiring) developers to prepare for job interviews with questions
 
 ## Install and Build
 
+### Developer Issues
```

Comment:
It should be an `h4` tag so there should be four pound-signs/hashtags. Also "issues" should not be capitalised to match other `h4`s.